algorithms/linfa-clustering/benches/gaussian_mixture.rs

```diff
@@ -46,9 +48,15 @@ fn gaussian_mixture_bench(c: &mut Criterion) {
     benchmark.finish();
 }
 
+#[cfg(not(target_os = "windows"))]
 criterion_group! {
-  name = benches;
-  config = Criterion::default();
+  name = not_win_benches;
+  config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
   targets = gaussian_mixture_bench
 }
-criterion_main!(benches);
+#[cfg(target_os = "windows")]
+criterion_group!(win_benches, gaussian_mixture_bench);
```

Comment:
If you give these two groups the same name, then you won't need to use any `cfg` below. Also insert a space after this line for readability.

Reply:
I meant:
```rust
#[cfg(not(target_os = "windows"))]
criterion_group! {
    name = benches;
    config = Criterion::default().with_profiler(PProfProfiler::new(100, Output::Flamegraph(None)));
    targets = appx_dbscan_bench
}
#[cfg(target_os = "windows")]
criterion_group!(benches, appx_dbscan_bench);
```

Reply:
ooo seems I just typod a lot and was mindless when I tested this bit. Making the change now.